lib/utils.js

```diff
@@ -8,7 +8,7 @@ const uuidv4 = require('uuid/v4');
 const Promise = require('./promise');
 const operators  = require('./operators');
 const operatorsArray = _.values(operators);
-
+const toBinaryUUID = require('binary-uuid').toBinaryUUID;
```

Comment:
Is it really worth it adding yet another dependency for a very niche data type like this?
Wouldn't it make more sense to split special datatypes and dialects into smaller packages?

Reply:
The dependency was written for this PR specifically - just left it as its own package as it was small and could be used in other places easily.